docs/migration/css-changes-in-version-4.md

```diff
@@ -5,3 +5,85 @@ title: Changes to Styles in 4.0
 ## Changes in Checkout Components
 
 * `cx-product-variants` selector has been moved to corresponding feature-lib `@spartacus/product`.
+
+## Change in Configurator Attribute Type Components
+
+* `cx-quantity` selector has been added to align its padding to the corresponding attribute type.
```

Comment:
Please re-phrase, don't know what's exactly meant

Reply:
Do you have a better idea? I don't :(

Reply:
Done!